resources/asciidoctor/lib/change_admonishment/extension.rb

```diff
@@ -0,0 +1,78 @@
+require 'asciidoctor/extensions'
+
+include Asciidoctor
+
+##
+# Extensions for marking when something was added, when it *will* be added, or
+# when it was deprecated.
+#
+# Usage
+#
+#   added::[6.0.0-beta1]
+#   coming::[6.0.0-beta1]
+#   deprecated::[6.0.0-beta1]
+#   Foo added:[6.0.0-beta1]
+#   Foo coming:[6.0.0-beta1]
+#   Foo deprecated:[6.0.0-beta1]
+#
+class ChangeAdmonishment < Extensions::Group
+  def activate registry
+    [
+        [:added, 'added'],
+        [:coming, 'changed'],
+        [:deprecated, 'deleted'],
```

Comment:
One of these things is not like the others! I'm sure there is a reason BUT, to people (like me!) reading this, the word `deprecated` _absolutely does not_ mean the same thing as `deleted`, in fact it very specifically implies "definitely totally 100% still present".

Is this dictated by the docbook side? I see from the tests that the revisionFlag will contain `deleted`. If that is within our control it may be worth its own issue? I have no clear sense of how much inertia there is behind it either.

Reply:
It is dictated by the docbook side, yeah. I'll file an issue for it! I think it is a thing that we can clean up now, but I'd like to break it into its own thing.

Reply:
https://github.com/elastic/docs/issues/588